docs/user-guide/release-notes/2.7.x.rst

```diff
@@ -60,6 +60,13 @@ Deprecation
 Upgrade Instructions for 2.6.x --> 2.7.0
 -----------------------------------------
 
+.. warning::
+
+   The default 'server_name' is the fully qualified domain name (FQDN). The CN in the certificate
```

Comment:
I think this should be rewritten to be clearer about what is changing with this release. I'm suggesting the following rewrite:

```
This release adjusts the default 'server_name' in `/etc/pulp/server.conf` to be the fully qualified domain name (FQDN), but previous releases on some platforms may have used the short hostname. Similarly, this release adjusts the pulp-gen-ca-certificate tool to use the FQDN as the `CN` in a certificate it generates.

As a result of this change, you may experience a hostname mismatch when you run Pulp. If you experience that issue, you can either set the value of 'server_name' in /etc/pulp/server.conf or regenerate your certificate using pulp-gen-ca-certificate.
```